### ISSUE TITLE
ES6 Array.prototype.findIndex() not found

### ISSUE THREAD

**Issue by SirRade** [palantir/tslint#3101]
### Bug Report

- __TSLint version__: 5.5.0
- __TypeScript version__: Version 2.4.2
- __Running TSLint via__: CLI (`$ tslint --type-check -p tslint.json 'src/**/*.ts'`)

#### TypeScript code being linted

```ts
let foo: number[] = [1, 2, 3]
foo.findIndex((item) => item == 2)
```

with `tslint.json` configuration:

```json
{
    "extends": [],
    "jsRules": {},
    "rules": {},
    "rulesDirectory": []
}
```

#### Actual behavior
```
Error at src/index.ts:2:5: Property 'findIndex' does not exist on type 'number[]'.
error Command failed with exit code 1
```
#### Expected behavior
Since this function is in ES6 and I've set my tsc target to es6, this should work.  
Is there some ES6 setting in TSLint that I should have activated?  
Is there a way to ignore this error?

**Comment by adidahiya** [comment]
You can ignore the error by removing the `--type-check` flag. It is no longer required and all it does is emit type checking errors in TSLint's output.

Have you also changed any `lib` options in `tsconfig.json`?

**Comment by SirRade** [comment]
Thank you, removing the flag worked without a problem :)  
The `lib` options where ommited.

**Comment by danm** [comment]
having the same problem here https://github.com/Microsoft/vscode/issues/63502
